CHANGELOG.md

```diff
@@ -5,11 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
The Makefile has hardcoded 0.11.0 version here:
```
MAJOR := 0
MINOR := 11
PATCH := 0
```

Also README and examples are using 0.11.0, not much important, but could be changed to 0.12.0. (do a search all for 0.11.0).